ironfish/src/rpc/routes/wallet/exportAccount.test.ts

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I missed that this is still in here, worth it to remove you think @jowparks ?

Reply:
Good call, I fixed it up, along with some of the import stuff to make the types live in the right spot